actions/app.go

```diff
@@ -44,6 +45,18 @@ var (
 // `ServeFiles` is a CATCH-ALL route, so it should always be
 // placed last in the route declarations, as it will prevent routes
 // declared after it to never be called.
+
+// @title          Paphos API
+// @version        1.0
+// @description    Base backend API to serve Pygamilion UI
+// @termsOfService http://swagger.io/terms/
+
+// @license.name   AGPL 3.0
+// @license.url    http://www.apache.org/licenses/LICENSE-2.0.html
```

Comment:
Minor oversight: license URL is still Apache 2.0.

Reply:
Ah right

Will use this instead: https://www.gnu.org/licenses/agpl-3.0.en.html